src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = Lottery.getRandomBall();
+        }
+
+        for (Ball ball : balls) {
+            System.out.println(ball.toString());
```

Comment:
toString() is called under the hood
```suggestion
            System.out.println(ball);
```